nixos/modules/services/amqp/rabbitmq.nix

```diff
@@ -210,8 +210,7 @@ in
 
       preStart = ''
         ${optionalString (cfg.cookie != "") ''
-            echo -n ${cfg.cookie} > ${cfg.dataDir}/.erlang.cookie
-            chmod 600 ${cfg.dataDir}/.erlang.cookie
+          install -m 600 <(echo -n ${cfg.cookie}) ${cfg.dataDir}/.erlang.cookie
```

Comment:
`cfg.cookie` is in the store anyway, by being part of the `preStart` script, so this creates a false sense of security.

Reply:
you are perfectly right.

do you suggest closing this PR? or rename the commit?